docs/API.md

```diff
@@ -72,22 +75,27 @@ The following table shows the current [JSON RPC API](https://eth.wiki/json-rpc/A
 | eth_getWork                                | Yes          |                                            |
 | eth_submitWork                             | Yes          |                                            |
 |                                            |              |                                            |
-| eth_subscribe                              | -            | not yet implemented                        |
-| eth_unsubscribe                            | -            | not yet implemented                        |
+| eth_subscribe                              | -            | not yet implemented (WebSockets only)      |
+| eth_unsubscribe                            | -            | not yet implemented (WebSockets only)      |
 |                                            |              |                                            |
 | engine_newPayloadV1                        | Yes          |                                            |
+| engine_newPayloadV2                        | -            | not yet implemented                        |
 | engine_forkchoiceUpdatedV1                 | Yes          |                                            |
+| engine_forkchoiceUpdatedV2                 | -            | not yet implemented                        |
 | engine_getPayloadV1                        | Yes          |                                            |
+| engine_getPayloadV2                        | -            | not yet implemented                        |
 | engine_exchangeTransitionConfigurationV1   | Yes          |                                            |
 |                                            |              |                                            |
 | debug_accountRange                         | Yes          |                                            |
+| debug_accountAt                            | -            | not yet implemented                        |
 | debug_getModifiedAccountsByHash            | Yes          |                                            |
 | debug_getModifiedAccountsByNumber          | Yes          |                                            |
 | debug_storageRangeAt                       | Yes          |                                            |
 | debug_traceBlockByHash                     | Yes          |                                            |
 | debug_traceBlockByNumber                   | Yes          |                                            |
-| debug_traceTransaction                     | Yes          |                                            |
```

Comment:
Saying 'uses JSON streaming' is a short but not fully correct way to describe the actual implementation, that's based on http chunked transfer encoding (see [https://en.wikipedia.org/wiki/Chunked_transfer_encoding]). 
May be we could state:

> uses chunked transfer encoding in http reply

 ?

Reply:
You may have a point here but I would prefer to use `JSON streaming` because it matches the terminology used in Erigon

Reply:
OK